CMakeLists.txt

```diff
@@ -0,0 +1,141 @@
+cmake_minimum_required(VERSION 3.0.2)
+set(CMAKE_DISABLE_SOURCE_CHANGES ON) # Must go before project() below
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON) # Must go before project() below
+
+if(CMAKE_VERSION VERSION_GREATER "3.12" OR POLICY CMP0074)
+  # Needed for THIRDPARTY_DIRECTORY support; this will be enabled by default
+  # once the minimum CMake version is at least 3.12.
+  cmake_policy(SET CMP0074 NEW)
+endif()
+
+# Figure out the version
+file(STRINGS "setup.cfg" _version REGEX "^version = ")
```

Comment:
Best I can do is `[\\t ]*` since `\s` [isn't included in CMake's regex dialect](https://cmake.org/cmake/help/v3.4/command/string.html#regex-specification).